### ISSUE TITLE
--sv option fails on v2.2.0 but runs on older versions

### ISSUE THREAD

**Issue by graemefox** [epi2me-labs/wf-human-variation#183]
### Operating System

Other Linux (please specify below)

### Other Linux

Ubuntu 20.04

### Workflow Version

v2.2.0

### Workflow Execution

Command line (Cluster)

### Other workflow execution

_No response_

### EPI2ME Version

_No response_

### CLI command run

```
nextflow pull epi2me-labs/wf-human-variation
Checking epi2me-labs/wf-human-variation ...
 Already-up-to-date - revision: 5651930a05 [master]

nextflow run epi2me-labs/wf-human-variation  -r master --bam NUH0001_1_P1_sup_5mCG_5hmCG.hg38.bam --ref /data/refs/hg38_simple.fa --sample_name 'NUH0001_hv_2.2.0' -profile standard --mod --snp --cnv --sv -resume --out_dir NUH0001_2.2.0_output
```

### Workflow Execution - CLI Execution Profile

standard (default)

### What happened?

Hello!
I'm running into issues running wf-human-variation since the release of v2.2.0; specifically the --sv module is causing an error.

My command above gives  the log output below. I can run the pipeline successfully if I leave out the --sv flag, and also if I use --sv with revision v2.1.0. The demo data also runs without any issues, --sv included.

Many thanks

### Relevant log output

```shell
nextflow pull epi2me-labs/wf-human-variation
Checking epi2me-labs/wf-human-variation ...
 Already-up-to-date - revision: 5651930a05 [master]

N E X T F L O W  ~  version 23.04.3                                                                                                                                                                                                           
Launching `https://github.com/epi2me-labs/wf-human-variation` [infallible_poitras] DSL2 - revision: 5651930a05 [master]                                                                                                                       
                                                                                                                                                                                                                                              
||||||||||   _____ ____ ___ ____  __  __ _____      _       _                                                                                                                                                                                 
||||||||||  | ____|  _ \_ _|___ \|  \/  | ____|    | | __ _| |__  ___                                                                                                                                                                         
|||||       |  _| | |_) | |  __) | |\/| |  _| _____| |/ _` | '_ \/ __|                                                                                                                                                                        
|||||       | |___|  __/| | / __/| |  | | |__|_____| | (_| | |_) \__ \                                                                                                                                                                        
||||||||||  |_____|_|  |___|_____|_|  |_|_____|    |_|\__,_|_.__/|___/                                                                                                                                                                        
||||||||||  wf-human-variation v2.2.0-g5651930                                                                                                                                                                                                
--------------------------------------------------------------------------------                                                                                                                                                              
Core Nextflow options                                                                                                                                                                                                                         
  revision       : master                                                                                                                                                                                                                     
  runName        : infallible_poitras                                                                                                                                                                                                         
  containerEngine: docker                                                                                                                                                                                                                     
  container      : ontresearch/wf-human-variation:sha0f033fe23a576cf4da9df8d2d6745d63329a8611                                                                                                                                                 
  launchDir      : /data/graeme                                                                                                                                                                                                               
  workDir        : /data/graeme/work                                                                                                                                                                                                          
  projectDir     : /home/mbzgf/.nextflow/assets/epi2me-labs/wf-human-variation                                                                                                                                                                
  userName       : mbzgf                                                                                                                                                                                                                      
  profile        : standard                                                                                                                                                                                                                   
  configFiles    : /home/mbzgf/.nextflow/assets/epi2me-labs/wf-human-variation/nextflow.config                                                                                                                                                
                                                                                                                                                                                                                                              
Workflow Options                                                                                                                                                                                                                              
  sv             : true                                                                                                                                                                                                                       
  snp            : true                                                                                                                                                                                                                       
  cnv            : true                                                                                                                                                                                                                       
  mod            : true                                                                                                                                                                                                                       
                                                                                                                                                                                                                                              
Main options                                                                                                                                                                                                                                  
  sample_name    : NUH0001_hv_2.2.0                                                                                                                                                                                                           
  bam            : NUH0001_1_P1_sup_5mCG_5hmCG.hg38.bam                                                                                                                                                                                       
  ref            : /data/refs/hg38_simple.fa                                                                                                                                                                                                  
  out_dir        : NUH0001_2.2.0_output                                                                                                                                                                                                       
                                                                                                                                                                                                                                              
!! Only displaying parameters that differ from the pipeline defaults !!                                                                                                                                                                       
--------------------------------------------------------------------------------                                                                                                                                                              
If you use epi2me-labs/wf-human-variation for your analysis please cite:                                                                                                                                                                      
                                                                                                                                                                                                                                              
* The nf-core framework                                                                                                                                                                                                                       
  https://doi.org/10.1038/s41587-020-0439-x                                                                                                                                                                                                   
                                                                                                                                                                                                                                              
                                                                                                                                                                                                                                              
--------------------------------------------------------------------------------                                                                                                                                                              
This is epi2me-labs/wf-human-variation v2.2.0-g5651930.                                                                                                                                                                                       
--------------------------------------------------------------------------------                                                                                                                                                              
Searching input for [.bam, .ubam] files.                                                                                                                                                                                                      
[-        ] process > ingress:checkBamHeaders               -                                                                                                                                                                                 
[-        ] process > ingress:sortBam                       -                                                                                                                                                                                 
[-        ] process > ingress:mergeBams                     -                                                                                                                                                                                 
[-        ] process > ingress:catSortBams                   -
[-        ] process > ingress:validateIndex                 -                                                                                                                                                                       [265/1825]
[-        ] process > ingress:samtools_index                -
[-        ] process > ingress:check_for_alignment           -
[-        ] process > ingress:minimap2_alignment            -
[-        ] process > getGenome                             -
[-        ] process > cram_cache                            -
[-        ] process > getAllChromosomesBed                  -
[-        ] process > mosdepth_input                        -
executor >  local (1)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
executor >  local (2)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
executor >  local (2)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
executor >  local (2)                                                                                                                                                                                                               [211/1825]
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
executor >  local (2)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -
[-        ] process > ingress:catSortBams                   -                                                           
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
executor >  local (2)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -                                                           
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
executor >  local (2)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -                                                           
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔                                                                                                                                              [156/1825]
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
executor >  local (2)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -                                                           
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
executor >  local (2)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -                                                           
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
[4f/a3c594] process > infer_sex (1)                         [100%] 1 of 1, cached: 1 ✔
[1e/943759] process > makeAlignmentReport                   [100%] 1 of 1, cached: 1 ✔
[-        ] process > failedQCReport                        -
[18/aab4ed] process > lookup_clair3_model (1)               [100%] 1 of 1, cached: 1 ✔                                  
[54/880c37] process > snp:make_chunks (1)                   [100%] 1 of 1, cached: 1 ✔
[b7/471eaa] process > snp:pileup_variants (632)             [100%] 632 of 632, cached: 632 ✔
[24/b58a1f] process > snp:aggregate_pileup_variants (1)     [100%] 1 of 1, cached: 1 ✔
[bd/e7ba6d] process > snp:select_het_snps (10)              [100%] 25 of 25, cached: 25 ✔
[db/25f9c0] process > snp:phase_contig (23)                 [100%] 25 of 25, cached: 25 ✔
[2c/292d31] process > snp:get_qual_filter (1)               [100%] 1 of 1, cached: 1 ✔
[93/14c034] process > snp:create_candidates (23)            [100%] 25 of 25, cached: 25 ✔
[b8/f05ad0] process > snp:evaluate_candidates (1345)        [100%] 1347 of 1347, cached: 1347 ✔
[a8/4f95d9] process > snp:aggregate_full_align_variants (1) [100%] 1 of 1, cached: 1 ✔
[26/e8641c] process > snp:merge_pileup_and_full_vars (20)   [100%] 25 of 25, cached: 25 ✔
[2d/8eae9d] process > snp:aggregate_all_variants (1)        [100%] 1 of 1, cached: 1 ✔                                                                                                                                              [101/1825]
[39/bbde85] process > sv:variantCall:sniffles2 (1)          [  0%] 0 of 1
[-        ] process > sv:variantCall:filterCalls            -
[-        ] process > sv:variantCall:sortVCF                -
[-        ] process > sv:annotate_sv_vcf                    -
[f8/940a1d] process > sv:runReport:getVersions              [100%] 1 of 1, cached: 1 ✔                                  
[8d/2a5728] process > sv:runReport:getParams                [100%] 1 of 1, cached: 1 ✔
[-        ] process > sv:runReport:report                   -
[-        ] process > output_sv                             -
[-        ] process > refine_with_sv                        -
[-        ] process > concat_refined_snp                    -
[-        ] process > annotate_snp_vcf                      -
[-        ] process > concat_snp_vcfs                       -
[-        ] process > sift_clinvar_snp_vcf                  -
[-        ] process > vcfStats                              -
[ff/5324f5] process > report_snp:getVersions                [100%] 1 of 1, cached: 1 ✔
[bd/0a06ea] process > report_snp:getParams                  [100%] 1 of 1, cached: 1 ✔
[-        ] process > report_snp:makeReport                 -
[-        ] process > output_snp                            -
[a6/dd2ec9] process > validate_modbam (1)                   [100%] 1 of 1, cached: 1 ✔
[01/ff5323] process > sample_probs (1)                      [100%] 1 of 1, cached: 1 ✔
[ab/1aaf09] process > mod:modkit (9)                        [100%] 25 of 25, cached: 25 ✔                               
[0d/329d01] process > mod:concat_bedmethyl (1)              [100%] 1 of 1, cached: 1 ✔
[8c/89de94] process > cnv_spectre:mosdepth (1)              [100%] 1 of 1, cached: 1 ✔
[ff/e6ad53] process > cnv_spectre:callCNV (1)               [100%] 1 of 1, cached: 1 ✔
[7d/244ec5] process > cnv_spectre:bgzip_and_index_vcf (1)   [100%] 1 of 1, cached: 1 ✔
[e1/5cffe8] process > cnv_spectre:annotate_vcf (1)          [100%] 1 of 1, cached: 1 ✔
[d2/dab040] process > cnv_spectre:getVersions               [100%] 1 of 1, cached: 1 ✔
[ab/df1b9d] process > cnv_spectre:add_snp_tools_to_versions [100%] 1 of 1, cached: 1 ✔
[b6/0fdf88] process > cnv_spectre:getParams                 [100%] 1 of 1, cached: 1 ✔
[8b/5e54bd] process > cnv_spectre:makeReport (1)            [100%] 1 of 1, cached: 1 ✔
[6a/8dd6d1] process > output_cnv (2)                        [100%] 2 of 2, cached: 2 ✔
[be/b047b7] process > configure_jbrowse (1)                 [  0%] 0 of 1
[-        ] process > combine_metrics_json                  -
[8b/290823] process > publish_artifact (8)                  [100%] 8 of 8, cached: 8
Autoselected Clair3 model: r1041_e82_400bps_sup_v410
executor >  local (2)                                       
[f0/f3261d] process > ingress:checkBamHeaders (1)           [100%] 1 of 1, cached: 1 ✔                                  
[-        ] process > ingress:sortBam                       -
[-        ] process > ingress:mergeBams                     -
[-        ] process > ingress:catSortBams                   -
[31/171d18] process > ingress:validateIndex (1)             [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:samtools_index                -
[d7/8294af] process > ingress:check_for_alignment (1)       [100%] 1 of 1, cached: 1 ✔
[-        ] process > ingress:minimap2_alignment            -
[c2/df09af] process > getGenome (1)                         [100%] 1 of 1, cached: 1 ✔
[ad/6872c2] process > cram_cache (1)                        [100%] 1 of 1, cached: 1 ✔                                  
[36/98c5cf] process > getAllChromosomesBed (1)              [100%] 1 of 1, cached: 1 ✔                                  
[c1/74dc5f] process > mosdepth_input (1)                    [100%] 1 of 1, cached: 1 ✔
[f2/6c5975] process > getVersions                           [100%] 1 of 1, cached: 1 ✔
[7b/e98383] process > getParams                             [100%] 1 of 1, cached: 1 ✔
[61/01d505] process > readStats (1)                         [100%] 1 of 1, cached: 1 ✔
[4f/a3c594] process > infer_sex (1)                         [100%] 1 of 1, cached: 1 ✔
[1e/943759] process > makeAlignmentReport                   [100%] 1 of 1, cached: 1 ✔
[-        ] process > failedQCReport                        -
[18/aab4ed] process > lookup_clair3_model (1)               [100%] 1 of 1, cached: 1 ✔                                                                                                                                               [46/1825]
[54/880c37] process > snp:make_chunks (1)                   [100%] 1 of 1, cached: 1 ✔
[b7/471eaa] process > snp:pileup_variants (632)             [100%] 632 of 632, cached: 632 ✔
[24/b58a1f] process > snp:aggregate_pileup_variants (1)     [100%] 1 of 1, cached: 1 ✔
[bd/e7ba6d] process > snp:select_het_snps (10)              [100%] 25 of 25, cached: 25 ✔                               
[db/25f9c0] process > snp:phase_contig (23)                 [100%] 25 of 25, cached: 25 ✔
[2c/292d31] process > snp:get_qual_filter (1)               [100%] 1 of 1, cached: 1 ✔
[93/14c034] process > snp:create_candidates (23)            [100%] 25 of 25, cached: 25 ✔
[b8/f05ad0] process > snp:evaluate_candidates (1345)        [100%] 1347 of 1347, cached: 1347 ✔
[a8/4f95d9] process > snp:aggregate_full_align_variants (1) [100%] 1 of 1, cached: 1 ✔
[26/e8641c] process > snp:merge_pileup_and_full_vars (20)   [100%] 25 of 25, cached: 25 ✔
[2d/8eae9d] process > snp:aggregate_all_variants (1)        [100%] 1 of 1, cached: 1 ✔
[39/bbde85] process > sv:variantCall:sniffles2 (1)          [100%] 1 of 1, failed: 1 ✘
[-        ] process > sv:variantCall:filterCalls            -
[-        ] process > sv:variantCall:sortVCF                -                                                           
[-        ] process > sv:annotate_sv_vcf                    -
[f8/940a1d] process > sv:runReport:getVersions              [100%] 1 of 1, cached: 1 ✔
[8d/2a5728] process > sv:runReport:getParams                [100%] 1 of 1, cached: 1 ✔
[-        ] process > sv:runReport:report                   -
[-        ] process > output_sv                             [  0%] 0 of 1
[-        ] process > refine_with_sv                        -
[-        ] process > concat_refined_snp                    -
[-        ] process > annotate_snp_vcf                      -
[-        ] process > concat_snp_vcfs                       -
[-        ] process > sift_clinvar_snp_vcf                  -
[-        ] process > vcfStats                              -
[ff/5324f5] process > report_snp:getVersions                [100%] 1 of 1, cached: 1 ✔
[bd/0a06ea] process > report_snp:getParams                  [100%] 1 of 1, cached: 1 ✔
[-        ] process > report_snp:makeReport                 -
[-        ] process > output_snp                            -
[a6/dd2ec9] process > validate_modbam (1)                   [100%] 1 of 1, cached: 1 ✔
[01/ff5323] process > sample_probs (1)                      [100%] 1 of 1, cached: 1 ✔
[ab/1aaf09] process > mod:modkit (9)                        [100%] 25 of 25, cached: 25 ✔
[0d/329d01] process > mod:concat_bedmethyl (1)              [100%] 1 of 1, cached: 1 ✔
[8c/89de94] process > cnv_spectre:mosdepth (1)              [100%] 1 of 1, cached: 1 ✔
[ff/e6ad53] process > cnv_spectre:callCNV (1)               [100%] 1 of 1, cached: 1 ✔
[7d/244ec5] process > cnv_spectre:bgzip_and_index_vcf (1)   [100%] 1 of 1, cached: 1 ✔
[e1/5cffe8] process > cnv_spectre:annotate_vcf (1)          [100%] 1 of 1, cached: 1 ✔
[d2/dab040] process > cnv_spectre:getVersions               [100%] 1 of 1, cached: 1 ✔
[ab/df1b9d] process > cnv_spectre:add_snp_tools_to_versions [100%] 1 of 1, cached: 1 ✔
[b6/0fdf88] process > cnv_spectre:getParams                 [100%] 1 of 1, cached: 1 ✔
[8b/5e54bd] process > cnv_spectre:makeReport (1)            [100%] 1 of 1, cached: 1 ✔
[6a/8dd6d1] process > output_cnv (2)                        [100%] 2 of 2, cached: 2 ✔
[-        ] process > configure_jbrowse (1)                 -
[-        ] process > combine_metrics_json                  [  0%] 0 of 1
[8b/290823] process > publish_artifact (8)                  [100%] 8 of 8, cached: 8
Autoselected Clair3 model: r1041_e82_400bps_sup_v410
WARN: Inferring genetic sex of sample as params.sex was not provided.
WARN: Found zero basecall_model in the input alignment header, falling back to the model provided with --basecaller_cfg: dna_r10.4.1_e8.2_400bps_sup@v4.1.0
WARN: Automatically selecting TR BED: hg38.trf.bed
ERROR ~ Error executing process > 'sv:variantCall:sniffles2 (1)'

Caused by:
  Process `sv:variantCall:sniffles2 (1)` terminated with an error exit status (1)
Caused by:
  Process `sv:variantCall:sniffles2 (1)` terminated with an error exit status (1)

Command executed:

  sniffles         --threads 4         --sample-id NUH0001_hv_2.2.0         --output-rnames         --minsvlen 30         --cluster-merge-pos 150         --input reads.bam         --reference hg38_simple.fa         --input-exclude-flags 2
308         --tandem-repeats ${WFSV_TRBED_PATH}/hg38.trf.bed                           --vcf NUH0001_hv_2.2.0.sniffles.vcf
  sed '/.:0:0:0:NULL/d' NUH0001_hv_2.2.0.sniffles.vcf > tmp.vcf
  mv tmp.vcf NUH0001_hv_2.2.0.sniffles.vcf

Command exit status:
  1

Command output:
  Running Sniffles2, build 2.0.7-epi2me
    Run Mode: call_sample
    Start on: 2024/05/23 15:02:07
    Working dir: /data/graeme/work/39/bbde85ed7ade2519dd0c81276b2aa0
    Used command: /home/epi2melabs/conda/bin/sniffles --threads 4 --sample-id NUH0001_hv_2.2.0 --output-rnames --minsvlen 30 --cluster-merge-pos 150 --input reads.bam --reference hg38_simple.fa --input-exclude-flags 2308 --tandem-repeats 
/data/wf_human_sv_trbed/hg38.trf.bed --vcf NUH0001_hv_2.2.0.sniffles.vcf
  ==============================
  Opening for reading: reads.bam

Command error:
  Running Sniffles2, build 2.0.7-epi2me
    Run Mode: call_sample
    Start on: 2024/05/23 15:02:07
    Working dir: /data/graeme/work/39/bbde85ed7ade2519dd0c81276b2aa0
    Used command: /home/epi2melabs/conda/bin/sniffles --threads 4 --sample-id NUH0001_hv_2.2.0 --output-rnames --minsvlen 30 --cluster-merge-pos 150 --input reads.bam --reference hg38_simple.fa --input-exclude-flags 2308 --tandem-repeats 
/data/wf_human_sv_trbed/hg38.trf.bed --vcf NUH0001_hv_2.2.0.sniffles.vcf
  ==============================
  Opening for reading: reads.bam
  Traceback (most recent call last):
    File "/home/epi2melabs/conda/bin/sniffles", line 616, in <module>
      Sniffles2_Main(config.from_cmdline(),processes)
    File "/home/epi2melabs/conda/bin/sniffles", line 130, in Sniffles2_Main
      contig_tandem_repeats=util.load_tandem_repeats(config.tandem_repeats,config.tandem_repeat_region_pad)
    File "/home/epi2melabs/conda/lib/python3.8/site-packages/sniffles/util.py", line 104, in load_tandem_repeats
      with open(filename,"r") as handle:
  FileNotFoundError: [Errno 2] No such file or directory: '/data/wf_human_sv_trbed/hg38.trf.bed'

Work dir:
  /data/graeme/work/39/bbde85ed7ade2519dd0c81276b2aa0

Tip: view the complete command output by changing to the process work dir and entering the command `cat .command.out`

 -- Check '.nextflow.log' file for details
```


### Application activity log entry

_No response_

### Were you able to successfully run the latest version of the workflow with the demo data?

yes

### Other demo data information

_No response_

**Comment by RenzoTale88** [comment]
@graemefox can you please try the new version of the workflow without the `-resume` option and report back if you still encounter the same issue?

**Comment by RenzoTale88** [comment]
Hi @graemefox thanks for confirming this. Could you confirm the docker command in the `.command.run` script within the work directory please?
```
grep docker /data/graeme/work/39/bbde85ed7ade2519dd0c81276b2aa0/.command.run
```

**Comment by graemefox** [comment]
That gives
```
docker rm $NXF_BOXID &>/dev/null || true
    docker kill $NXF_BOXID
    docker run -i --cpu-shares 4096 --memory 6144m -e "NXF_DEBUG=${NXF_DEBUG:=0}" -v /data:/data -v /home/mbzgf/.nextflow/assets/epi2me-labs/wf-human-variation:/home/mbzgf/.nextflow/assets/epi2me-labs/wf-human-variation -w "$PWD" --user $(id -u):$(id -g) --group-add 100 --name $NXF_BOXID ontresearch/wf-human-variation-sv:sha8f02fbfaea233fe4763f7c7142696efcadab2f4d /bin/bash -c "eval $(nxf_container_env); /bin/bash /data/graeme/work/39/bbde85ed7ade2519dd0c81276b2aa0/.command.run nxf_trace"
```

**Comment by SamStudio8** [comment]
@graemefox Apologies - this is our fault as 2.2.0 ships some data inside the container in `/data/` which is being shadowed by your inputs from `/data/`. We are prepping a patch release of the workflow for this week so I'll update the container and slip in a fix for this week.

**Comment by graemefox** [comment]
No worries @SamStudio8 thanks for figuring this one out!

**Comment by SamStudio8** [comment]
@graemefox This should be fixed in the hot-of-the-press v2.2.2 release - please let us know how you get on!

**Comment by graemefox** [comment]
Awsome. Just ran it again with 2.2.2 and it runs with no issues. Thanks again!
